### PR TITLE
Add a survey form GET API view

### PR DESCRIPTION
**Pull request by adam-thomas**
Returns the full details of the survey, including the fieldset nested structure, as per the API description.

@incuna/backend review please? :)

### CODE REVIEW

**Comment by wytrych** [comment]
Are you actually using the JSON in testing?

**Comment by adam-thomas** [comment]
Not yet, no.

File: surveys/models.py [hunk 1]
```diff
@@ -81,6 +81,9 @@ class Survey(models.Model):
     def __str__(self):
         return self.name
 
+    def get_api_url(self):
+        return drf_reverse('survey-form', kwargs={'pk': self.pk})
```
Comment (KevinEtienne) [inline surveys/models.py]: Where do you need `get_api_url`?

File: surveys/urls.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.conf.urls import url
+
+from . import views_api
+
+
+urlpatterns = [
+    url(r'^forms/(?P<pk>\d+)/$', views_api.SurveyView.as_view(), name='survey-form'),
```
Comment (KevinEtienne) [inline surveys/urls.py]: I think `angular` queries the urls without a `/` at the end and I think in general we make it optional with a `?`:
```python
    url(r'^forms/(?P<pk>\d+)/?$', views_api.SurveyView.as_view(), name='survey-form'),
```

File: surveys/models.py (hunk 1 above)
Comment (adam-thomas) [inline surveys/models.py]: We'll need it for the API description tests coming up soonish.  I figured I'd throw it in.

File: surveys/urls.py (hunk 2 above)
Comment (adam-thomas) [inline surveys/urls.py]: Will add the `?`, thanks :)

**Comment by adam-thomas** [comment]
Updated :)

**Pull request merged**